templates/workspaces/base/porter.yaml

```diff
@@ -89,6 +89,10 @@ parameters:
     type: string
     default: ""
     description: "The id of the application role WorkspaceResearcher in the identity provider"
+  - name: app_role_id_airlock_manager
```

Comment:
We'd need to bump the porter version to make sure this builds

Reply:
Yes you are right. I will test it again